core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -112,9 +137,37 @@
       + "full (new schema is backward and forward compatible with latest registered schema)";
   protected static final String MASTER_ELIGIBILITY_DOC = 
       "If true, this node can participate in master election. In a multi-colo setup, turn this off" 
-      + "for clusters in the slave data center.";                               
+      + "for clusters in the slave data center.";
+  protected static final String KAFKASTORE_SECURITY_PROTOCOL_DOC =
+      "The security protocol to use when connecting with Kafka, the underlying persistent storage. "
+      + "Values can be `PLAINTEXT` or `SSL`.";
+  protected static final String KAFKASTORE_SSL_TRUSTSTORE_LOCATION_DOC =
+      "The location of the SSL trust store file used when `kafka.security.protocol` is configured "
+      + "to `SSL`.";
+  protected static final String KAFKASTORE_SSL_TRUSTSTORE_PASSWORD_DOC =
+      "The password to access the trust store configured with `kafka.ssl.truststore.location`.";
+  protected static final String KAFKASTORE_SSL_KEYSTORE_LOCATION_DOC =
+      "The location of the SSL keystore file used when `kafka.security.protocol` is configured "
+      + "to `SSL` and Kafka requires SSL authentication.";
+  protected static final String KAFKASTORE_SSL_KEYSTORE_PASSWORD_DOC =
+      "The password to access the keystore configured with `kafka.ssl.keystore.location`.";
+  protected static final String KAFKASTORE_SSL_KEY_PASSWORD_DOC =
+      "The password of the key contained in the keystore configured with "
+      + "`kafka.ssl.keystore.location`.";
```

Comment:
Another reason for not doing it this way is that one doesn't have to configure the location at all if the certificates are included in the JDK (see https://github.com/apache/kafka/pull/312 for more background).